ansible/templates/minigraph_meta.j2

```diff
@@ -112,7 +112,40 @@
 {% endif %}
         </a:Properties>
       </a:DeviceMetadata>
-{% set idx = 0 %}
+{% if num_asics > 1 and switch_type is defined and switch_type == 'voq' %}
```

Comment:
switch_type is defined and == 'voq' seems to be repeating twice: 115 and 126

Reply:
Addressed in commit 7daf2e0